.gitignore

```diff
@@ -14,15 +14,16 @@
 .Spotlight-V100
 .Trashes
 # Icon must end with two \r
-Icon

+Icon
+
```

Comment:
Revert this line, it must end with two \r

Reply:
Lovely that my editor seems to do that automatically...